Tools/CMake/AMReXOptions.cmake

```diff
@@ -253,6 +253,18 @@ cmake_dependent_option( AMReX_GPU_RDC "Enable Relocatable Device Code" ${_GPU_RD
 unset(_GPU_RDC_default)
 print_option(AMReX_GPU_RDC)
 
+#
+# Fast Math    ================================================================
+#
+set(_FASTMATH_default OFF)
+if(AMReX_GPU_BACKEND STREQUAL CUDA)  # note: historic settings
+# if(NOT AMReX_GPU_BACKEND STREQUAL NONE)  # note: this would be more consistent for GPUs
+    set(_FASTMATH_default ON)
+endif()
+option(AMReX_FASTMATH  "Enable fast-math optimizations" ${_FASTMATH_default})
```

Comment:
This should have just been `OFF` and was an overlook to add, because have `AMReX_CUDA_FASTMATH` already. Fix in #4591